### PR TITLE
Add new Heading component

### PR DESCRIPTION
**Pull request by williamkolmacka**
This Pull Request meets the following criteria:

* [ ] Tests have been added/adjusted for my new feature
* [ ] New Components are registered in index.js of my project

Closes: #93

### CODE REVIEW

File: src/Heading/index.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+const sizeHeading = {
+  display: tokens.fontSizeHeadingDisplay,
+  large: tokens.fontSizeHeadingLarge,
+  medium: tokens.fontSizeHeadingMedium,
+  small: tokens.fontSizeHeadingSmall,
+};
+
+type Props = {
+  element: "h1" | "h2" | "h3" | "h4" | "h5" | "h6" | "div",
```
Comment (tomashapl) [inline src/Heading/index.js]: can you pls use only `h1` `h2` `h3` for now ? talked in person with @darkwindcz about

File: src/Heading/index.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  size: $Keys<typeof sizeHeading>,
+  weight: $Keys<typeof weightHeading>,
+  className?: string | void,
```
[the remark this answers is not in the document]
Comment (jaroslav-kubicek) [reply src/Heading/index.js]: `className?: string | void` doesn't make much sense, just `className?: string | void` is fine IMHO

[the remark this answers is not in the document]
Comment (williamkolmacka) [reply src/Heading/index.js]: Going to fix it.

File: src/Heading/index.js (hunk 1 above)
Comment (williamkolmacka) [reply src/Heading/index.js]: Yes, no problem.

**Pull request merged**